store/reducers/crypto.reducers.tsx

```diff
@@ -11,6 +11,13 @@ const initialState = {
 
 export const CryptoReducer = (state: State = initialState, action: Action): State => {
     switch (action.type) {
+        case ADD_CRYPTO:
+            const updateList = [...state.cryptoList, action.payload]
+            return {
+                ...state,
+                cryptoList: updateList
```

Comment:
[suggestion] Since there's not much need to separate the new array into a new variable, it would be better to declare the array here instead